<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GeneralSettingsForm</class>
 <widget class="QWidget" name="GeneralSettingsForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>358</width>
    <height>285</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ustawianie ogólnych parametrów</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>199</width>
     <height>28</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Podaj liczbę bębnów:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="reelQuantitySpinBox">
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>8</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>90</y>
     <width>200</width>
     <height>28</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Podaj liczbę symboli: </string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="symbolQuantitySpinBox">
      <property name="minimum">
       <number>1</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QFrame" name="frame_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>140</y>
     <width>281</width>
     <height>41</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="label_36">
          <property name="locale">
           <locale language="Polish" country="Poland"/>
          </property>
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Podaj liczbę symboli widocznych &lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_37">
          <property name="locale">
           <locale language="Polish" country="Poland"/>
          </property>
          <property name="text">
           <string>w pionie na ekranie</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QSpinBox" name="symbolsDisplayedVerticallyOnScreenQuantitySpinBox">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>5</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>210</y>
     <width>166</width>
     <height>25</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
